tests/unit/IOUUtilsTest.js

```diff
@@ -132,3 +146,45 @@ describe('isIOUReportPendingCurrencyConversion', () => {
     });
 });
 
+describe('getCurrencyDecimals', () => {
+    beforeAll(() => initCurrencyList());
+    test('Currency decimals smaller than or equal 2', () => {
+        expect(IOUUtils.getCurrencyDecimals('JPY')).toBe(0);
+        expect(IOUUtils.getCurrencyDecimals('USD')).toBe(2);
+    });
+
+    test('Currency decimals larger than 2 should return 2', () => {
+        // Actual: 3
+        expect(IOUUtils.getCurrencyDecimals('LYD')).toBe(2);
+
+        // Actual: 4
+        expect(IOUUtils.getCurrencyDecimals('UYW')).toBe(2);
+    });
+});
+
+describe('getCurrencyUnits', () => {
+    beforeAll(() => initCurrencyList());
+    test('Currency with decimals smaller than or equal 2', () => {
+        expect(IOUUtils.getCurrencyUnits('JPY')).toBe(1);
+        expect(IOUUtils.getCurrencyUnits('USD')).toBe(100);
+    });
+
+    test('Currency with decimals larger than 2 should be floor to 2', () => {
+        expect(IOUUtils.getCurrencyUnits('LYD')).toBe(100);
+    });
+});
+
+describe('calculateAmount', () => {
+    beforeAll(() => initCurrencyList());
+    test('103 JPY split among 3 participants including the default user should be [35, 34, 34]', () => {
+        const participants = ['tonystark@expensify.com', 'reedrichards@expensify.com'];
+        expect(IOUUtils.calculateAmount(participants, 103, 'JPY', true)).toBe(35);
+        expect(IOUUtils.calculateAmount(participants, 103, 'JPY')).toBe(34);
+    });
+
+    test('10 AFN split among 4 participants including the default user should be [1, 3, 3, 3]', () => {
+        const participants = ['tonystark@expensify.com', 'reedrichards@expensify.com', 'suestorm@expensify.com'];
+        expect(IOUUtils.calculateAmount(participants, 10, 'AFN', true)).toBe(1);
+        expect(IOUUtils.calculateAmount(participants, 10, 'AFN')).toBe(3);
+    });
```

Comment:
Can you add some tests for `calculateAmount` using currencies with decimals like USD? 🙏